extensions/authentication-passport/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright (c) IBM Corp. 2017,2019. All Rights Reserved.
```

Comment:
minor details. since it's created this year, shouldn't it be just `2019` for the year?